tests/system/model_monitoring/test_model_monitoring.py

```diff
@@ -90,12 +87,19 @@ def random_labels():
 class TestModelEndpointsOperations(TestMLRunSystemModelMonitoring):
     """Applying basic model endpoint CRUD operations through MLRun API"""
 
-    project_name = "mm-app-project-v6"
+    project_name = "mm-app-project"
 
     def setup_method(self, method):
         super().setup_method(method)
         if method.__name__ == "test_list_endpoints_without_creds":
             return
+        tag = None
+        if method.__name__ == "test_store_endpoint_update_existing":
+            tag = "latest"
```

Comment:
have to say there are better ways to do this without having to specify the test name but not urgent for now